types/react-transition-group/react-transition-group-tests.tsx

```diff
@@ -17,6 +17,7 @@ const Test: React.StatelessComponent = () => {
         <TransitionGroup
```

Comment:
I changed import to `import X = require("...")` so they are cjs imports as well as tranpiled code. I tested this in my actual aplications and that seems to work fine. Is that what you were asking? If not then could you share what is the proper solution here? The "namespace workaround" is all over the place in the definitions in DT and countless modules use such imports. Are you saying that they are all broken?